core/preferences/src/main/proto/settings.proto

```diff
@@ -225,9 +225,13 @@ message Settings {
     bool launch_on_enter = 4;
     enum SearchResultOrdering {
       Alphabetic = 0;
-      Relevance = 1;
+      TotalLaunchCount = 1;
+      RecentLaunchCount = 2;
+      Weighted = 3;
     }
     SearchResultOrdering search_result_ordering = 5;
+    int64 recent_launch_count_timespan_ms = 6;
+    double weight_factor = 7;
```

Comment:
There is no real benefit for storing this in user settings. It only makes it harder for us, in case we want to tweak that value in a future update.

Reply:
I intended these ones to be user-configurable. I was referring to an upper limit for `recent_launch_count_timespan_ms`, which would be used by the database for cleanup.

Like: `maxTimespanMs = 1 Month`, so the database would always keep 1 month of timestamps. And `recentLaunchCountTimespanMs` could be any value between one day and `maxTimespanMs`, which would be the value used for reordering.

The same goes for the weight-factor, if users wanted a faster change they could increase alpha, otherwise, decrease it.